docs/reference/child_admin.rst

```diff
@@ -66,7 +66,7 @@ class::
 
             if ($this->isGranted('LIST')) {
                 $menu->addChild('Manage Videos', [
-                    'uri' => $admin->generateUrl('sonata.admin.video.list', ['id' => $id])
+                    'uri' => $admin->generateUrl('App\Admin\VideoAdmin.list', ['id' => $id])
```

Comment:
Can you please add an information, how the `VideoAdmin` service is registered

Reply:
sure

Reply:
I hope this is what you expected